### PR TITLE
[WIP] Added login, signup, forget, profile pages

### PR DESCRIPTION
**Pull request by rakeshlanjewarwork**
Dear Team,
               I have added the login, signup, forget password,
               Please have a look at it.
               Template has been taken from this site :- [https://material-ui.com/getting-started/templates/](https://material-ui.com/getting-started/templates/)
               #hacktoberfest

### CODE REVIEW

**Comment by rakeshlanjewarwork** [comment]
> Thanks for the PR @rakeshlanjewar , We are using .js extension , so kindly stick to it.

Sorry to say but i am not able to understand to use .js extention , as i can see there are jsx extention

**Comment by rakeshlanjewarwork** [comment]
Do i need to remove material-ui?
We can redesign app with material design.
Which will + point for designing

File: src/components/UserLogin/userlogin.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+body {
+    background-color: '#fff'
+}
+
```
Comment (siddhant1) [inline src/components/UserLogin/userlogin.css]: Let's not add too generic selectors.

File: src/components/UserLogin/userlogin.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+.avatar {
+    margin: 8px;
+    background-color: #f50057;
+}
+
+.form {
+    width: 100%;
+    margin-top: 8px;
+}
```
Comment (siddhant1) [inline src/components/UserLogin/userlogin.css]: Make the generic  selector nested 
Like .sign-in .form{}

File: src/components/UserLogin/userlogin.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+.submit {
+    margin: 24px 0px 16px
+}
```
Comment (siddhant1) [inline src/components/UserLogin/userlogin.css]: Same as above

File: src/components/UserLogin/userlogin.css (hunk 2 above)
Comment (rakeshlanjewarwork) [reply src/components/UserLogin/userlogin.css]: ok

**Comment by siddhant1** [comment]
Thanks for the pr! Just one thing i , make sure you make pr from a separate branch so if you want to work on a different features the commits won't add up . 
Thanks for the help though , we will probably need to add some state to manage and integrate the back-end service , i am gonna open a separate issue for that .

**Comment by rakeshlanjewarwork** [comment]
I would like to work more on this project.
Please share the roadmap so i can work accoding to that

**Pull request merged**